lib/msf/core/exploit/remote/kerberos/ticket/storage/stored_ticket.rb

```diff
@@ -50,6 +51,8 @@ def expired?(now = Time.now)
       !(tkt_start < now && now < tkt_end)
     end
 
+    attr_reader :status, :realm, :client, :server
+
```

Comment:
```suggestion
```

Reply:
tried to remove this, I'm not using the :realm, :client, or :server but I do need to pluck out the status to display it in the klist table, you happy for me to just keep status here? along with `parse_loot_info` but just get rid of the rest?

Reply:
link to where I use it
https://github.com/rapid7/metasploit-framework/pull/17496/files#diff-536786fc96565668c5b709979a7aecfd2d0b1b59e8dd5d0cfa40dc8d6686906aR140-R148